lib/chef/guard_interpreter/resource_guard_interpreter.rb

```diff
@@ -68,7 +68,8 @@ def evaluate_action(action=nil, &block)
         run_action = action || @resource.action
 
         begin
-          @resource.run_action(run_action)
+          # Coerce to an array to be safe.
```

Comment:
Safe from what?  I think this comment is warranted, I just want to know `why` more fully